lib/remove_person.rb

```diff
@@ -0,0 +1,91 @@
+#!usr/bin/ruby
+
+puts "Enter name of person to remove:"
+name = gets.strip!
+puts "\n###########################\n# Removing #{name}\n###########################"
+
+def short_name(name)
+  name.gsub(" ", "-").downcase
+end
+
+def first_name(name)
+  name.split(" ")[0]
+end
+
+##########################################
+# Remove person from people.yml          #
+##########################################
+
+people_yml_path = "#{Dir.pwd}/data/people.yml"
+tmp_file_path = "#{people_yml_path}.tmp"
+found_person = false
+
+File.open(tmp_file_path, "w") do |output|
+  File.open(people_yml_path, "r") do |input|
+    to_remove_block = false
+    input.each_line do |line|
+      if to_remove_block && line.strip.empty?
```

Comment:
@NeilvB  Ah, sure that makes sense. I thought the syntax for separation actually required the blank line, but this sounds more robust.